### ISSUE TITLE
[req] GiroTorrent

### ISSUE THREAD

**Issue by garfield69** [Jackett/Jackett#6766]
GiroTorrent is an ITALIAN Private Torrent Tracker for MOVIES / TV / GENERAL
http://girotorrent.org/

**Comment by garfield69** [comment]
Duh, the indexer for this site already exists #3158

**Comment by garfield69** [comment]
http://girotorrent.org/ site is down currently displaying test page
![image](https://user-images.githubusercontent.com/6933240/91694772-3cd48e80-ebc1-11ea-82ef-36331647ffa5.png)

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
http://girotorrent.org/ returning
```
Table 'girotorr_xbtit.xbtit_settings' doesn't exist
```

**Comment by garfield69** [comment]
now returning
```
girotorrent.org
Site under maintenance due to update
```

**Comment by garfield69** [comment]
Update: can now login, but the torrent browser is still unavailable. Oh well, progress of sorts.

**Comment by garfield69** [comment]
back to full splash no login
```
girotorrent.org
Site under maintenance due to update


Sorry for the site delay, but this was a huge server and tracker update.
Keep checking here to see when we'll be back online.
```

**Comment by garfield69** [comment]
online.

**Comment by garfield69** [comment]
https://girotorrent.org/
```
Link opens in a new tab so when done come back here & refresh to continue.
Your license is invalid, please vaildate your license [Here](https://girotorrent.org/include/activate.php).
Only the owner of this site can activate the license so if your not the owner Please Do Not Try & contact the owner about this.


The site is under maintenance.
Do not click and do not enter your site credentials because it is a license update problem.
Please try again later or press F5 on your keyboard to refresh the page.
Thanks for your understanding.
```

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
27/oct/2023 per tutelare il progetto girotorrent  dal mese prossimo solo gli specialvip potranno scaricare
from next month, only VIP will be able to download.

**Comment by garfield69** [comment]
https://girotorrent.org/ and https://girotorrent.org/index.php?page=login both return http 200 but with empty HTML

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
Girotorrent have upgraded from xbtitFM 3.1.00 to  xbtitFM v4.1.33 
My credentials are not recognized and the forgot password does not know my email
I've had to register afresh.
Waiting on the confirmation email, nothing so far.

**Comment by garfield69** [comment]
Well, I never got the confirmation email, but when i tried the domain again I got the home page with the welcome back.
[late edit] And my profile shows I joined back on 2020 so i don't know whats happened.
Will check later today for any indexer changes that need doing.
[late edit] now on xbtitFM v4.2.12

**Comment by garfield69** [comment]
v0.24.790

**Comment by gabo988** [comment]
> Well, I never got the confirmation email, but when i tried the domain again I got the home page with the welcome back. [late edit] And my profile shows I joined back on 2020 so i don't know whats happened. Will check later today for any indexer changes that need doing. [late edit] now on xbtitFM v4.2.12

Hi, I currently get similar behavior, with credentials not recognized, forgot password page showing http error 500 and attempt to register again but not getting any confirmation mail. Any clue on how to unblock this? Is the site still working to you? Thank you

**Comment by garfield69** [comment]
>Is the site still working to you?

Yes I can access the site with my browser and the Jackett indexer is working too.

>Any clue on how to unblock this?

Perhaps use your browser to access the domain and then do a clear cookies and site data using the relevant browser option.
